internal/lock_test.go

```diff
@@ -5,13 +5,12 @@ package internal
 
 import (
 	"fmt"
+	"github.com/cisco-open/grabit/test"
```

Comment:
Why are you changing the ordering of the import here? There is a good reason why they are like they are:
- standard library imports first
- non standard lib import second